dashboard/modules/job/job_manager.py

```diff
@@ -469,6 +469,7 @@ async def submit_job(
         runtime_env: Optional[Dict[str, Any]] = None,
         metadata: Optional[Dict[str, str]] = None,
         _start_signal_actor: Optional[ActorHandle] = None,
+        _driver_on_current_node: bool = True,
```

Comment:
I think it should always be true. Any concerns?

Reply:
If we submit a job on JobAgent, `_driver_on_current_node` will be False. when the `JobAgent` completed `_driver_on_current_node` will be always `False`